js/libs/form.js

```diff
@@ -655,6 +655,34 @@ Fliplet.Widget.instance('form-builder', function(data) {
           $vm.saveProgress();
         }, 10);
 
+        function validateCheckboxValue(value, options) {
+          value = _.isArray(value) ? value : [value];
+
+          value = value.map(function(elem) {
+            return elem.trim();
+          });
+
+          if (options.length) {
+            var valueInOptions = [];
```

Comment:
Why are we setting up an empty array and then one-by-one pushing values into the array?

We should be able to use:

```suggestion
            var valueInOptions = _.filter(values, fn);
```

Reply:
@MaksymShokin ^

Reply:
fixed